core/templates/dev/head/pages/topic_viewer/topic_viewer.html

```diff
@@ -0,0 +1,155 @@
+{% extends 'pages/base.html' %}
+
+{% block maintitle %}
+  {{ topic_name }} - Oppia
+{% endblock maintitle %}
+
+{% block navbar_breadcrumb %}
+  <topics-viewer-navbar-breadcrumb topic-name="{{ topic_name }}">
+  </topics-viewer-navbar-breadcrumb>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <background-banner></background-banner>
+  <div ng-controller="TopicViewer" class="oppia-topic-viewer-container" style="position: relative; padding-top: 163px;">
```

Comment:
I think it might be better to wrap these (the stories list part) in another directive (maybe StoriesTilesDirective?) so that practice sessions can also be added as a separate directive here itself in the future.
Check topic_editor.html for reference.

Reply:
Added a new directive called StoriesTilesDirective.